src/main/java/uk/gov/justice/digital/delius/controller/AuthenticationController.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.delius.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.justice.digital.delius.data.api.UserDetails;
+import uk.gov.justice.digital.delius.service.UserService;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+public class AuthenticationController {
+
+    private final UserService userService;
+
+    public AuthenticationController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @RequestMapping("/authenticate")
+    public ResponseEntity authenticate(final @RequestParam String username, @RequestParam String password) {
```

Comment:
Is this called using HTTPS or HTTP? Just thinking that the password in the body will not be encrypted in flight over HTTP, so visible in logs (and packet sniffers).  Do all clients use the Tolomy HTTPS gateway to access the community API? Its fine if they do.

Reply:
All http within UKCloud I'm afraid. Along with all sensitive probation data. So from client to auth app it would be https, then from MoJ AWS Cloud platform to any UKCLoud gateway would be https but once in side UKCloud it would be plain text. The same as Delius now

Reply:
Actually even from the browser client to Delius in probation is plain http since they are on a "secure"  VPN . So believe it or not Delius uses http in live. Presumably this will all change when it is delius is migrated to AWS.

Reply:
Although in pre-prod it is SSL but with and invalid certificate. But prod is plain http